From: Wolfgang Rosenauer <stark@suse.de>
Subject: disable visibility feature
References:
https://bugzilla.novell.com/show_bug.cgi?id=83908
http://gcc.gnu.org/bugzilla/show_bug.cgi?id=20297

because of a gcc bug it's not possible to build with that
feature at the moment

Index: configure
===================================================================
--- configure.orig
+++ configure
@@ -7949,7 +7949,7 @@ EOF
                       if ${CC-cc} -Werror -S conftest.c -o conftest.s >/dev/null 2>&1; then
                         if egrep '\.(hidden|extern_private).*foo_hidden' conftest.s >/dev/null; then
                           if ! egrep '\.(hidden|extern_private).*foo_default' conftest.s > /dev/null; then
-                            ac_cv_visibility_pragma=yes
+                            ac_cv_visibility_pragma=no
                           fi
                         fi
                       fi
